Testscripts/Windows/NET-Add-NIC-After-Provision.ps1

```diff
@@ -49,8 +49,14 @@ function Main {
 			# Add a new network interface
 			$ipConfig = New-AzNetworkInterfaceIpConfig -Name $ipConfigName -PrivateIpAddressVersion `
 				IPv4 -PrivateIpAddress $ipAddr -SubnetId $vnet.Subnets[0].Id
-			$nic = New-AzNetworkInterface -Name $nicName -ResourceGroupName $AllVMData.ResourceGroupName `
-				-Location $AllVMData.Location -IpConfiguration $ipConfig -Force
+			if ($CurrentTestData.SetupConfig.Networking -eq 'SRIOV') {
+				$nic = New-AzNetworkInterface -Name $nicName -ResourceGroupName $AllVMData.ResourceGroupName `
```

Comment:
@juhlee-microsoft  this change would add the SRIOV Extra NICs as expected. Thank you @harz566 !

Reply:
Sorry. My comment went to the wrong way.  I meant SR-IOV nic definition inside of XML files for the rest of test cases that we plan for next Sprint. This is correct lines for SR-IOV nic setup.